modules/ownKeys.js

```diff
@@ -0,0 +1,19 @@
+import objectKeys from 'object-keys'
+
+const ownKeys = (object) => {
+  if (typeof Reflect === 'object' && typeof Reflect.ownKeys === 'function') {
+    return Reflect.ownKeys(object)
+      .filter(key => Object.getOwnPropertyDescriptor(object, key).enumerable)
+  }
+
+  if (typeof Object.getOwnPropertySymbols === 'function') {
+    return Object.getOwnPropertySymbols(object)
+      .filter(key => Object.getOwnPropertyDescriptor(object, key).enumerable)
+      .concat(objectKeys(object))
```

Comment:
one more thing :-) `Reflect.ownKeys`, by spec, returns string keys first, and then symbol keys. I think you want to flip the concat order here (even though it's not likely to affect its use in `toInclude`, it's good to make a generic `ownKeys` be correct)